Helper/Customer.php

```diff
@@ -69,8 +79,11 @@ public function getAddressVarsByCustomer(MageCustomer $customer)
     public function getCustomVariables($object, $data = [])
     {        
         $dataModel = $object->getDataModel();
-        $store = $this->storeManager->getStore($dataModel->getStoreId());
+        $storeId = $dataModel->getStoreId();
+        $store = $this->storeManager->getStore($storeId);
         $group = $this->customerGroupModel->load($dataModel->getGroupId());
+        $case = $this->sailthruSettings->getSelectCase($storeId);
+        $vars = $this->sailthruVars->getNameKeys($case);
```

Comment:
This doesn't follow the pattern you have below with `$nameVars`

But also, it may end up extending beyond name vars. Can we name the result var more generically? e.g. `varKeys` or something?